### PR TITLE
Thread of coherence in presence section

### PR DESCRIPTION
**Pull request by chhoumann**
This is a proposal to create a better thread of coherence in the presence chapter.
It introduces the categories we examine in the introductory section, and then the technologies are described in the subsequent chapters.
Each category section is a text introducing the category, before going in-depth with a technology.

image

- proposal - thread of coherence in presence chapter
- rename presence intro
- remove sprint 1 from index

### CODE REVIEW

File: report/src/bib/mybib.bib [hunk 1]
```diff
@@ -398,3 +398,22 @@ @article{longoAccurateOccupancyEstimation2019
...
+  langid       = {english},
+  keywords     = {📖,📚,Building automation system,Controller robustness,Heating; ventilation and air-conditioning,Indoor air quality,Information deficiencies,Model predictive control,Occupancy estimations,Office buildings,Predictive control},
```
Comment (Pattrigue) [inline report/src/bib/mybib.bib:417]: why emoji

File: report/src/bib/mybib.bib [hunk 2]
```diff
@@ -398,3 +398,22 @@ @article{longoAccurateOccupancyEstimation2019
...
+  file         = {C\:\\Users\\chhou\\Zotero\\storage\\SYQG4KTY\\Gruber et al_2014_CO2 sensors for occupancy estimations.pdf;C\:\\Users\\chhou\\Zotero\\storage\\5T3W33S8\\S0378778814006914.html}
+}
```
Comment (Pattrigue) [inline report/src/bib/mybib.bib:419]:
```suggestion
}
```

File: report/src/presence_technologies/index.tex [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+% \input{presence_env.tex}
+
+\section{Video-based systems}\label{sec:presence_video}
+% Intro text
+\todo{Insert intro text here to video-based systems.}
+% \input{presence_video.tex}
+
+\section{Radio-based systems}\label{sec:presence_radio}
+Radio-based occupancy detection systems make use of wireless communication technologies, such as Wi-Fi and Bluetooth, to determine the presence of individuals in a given area.
+These systems exploit the fact that most people carry devices, like smartphones and wearables, that emit wireless signals.
```
Comment (Pattrigue) [inline report/src/presence_technologies/index.tex:16]:
```suggestion
These systems exploit the fact that most people carry devices that emit wireless signals like smartphones and wearables.
```
Interposed sentence could be removed?

File: report/src/presence_technologies/index.tex [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+By monitoring the strength and distribution of these signals, radio-based systems can infer the number of occupants and their locations within the monitored space.\cite{longoAccurateOccupancyEstimation2019, teissedre-2019}
```
Comment (Pattrigue) [inline report/src/presence_technologies/index.tex:17]:
```suggestion
By monitoring the strength and distribution of these signals, radio-based systems can infer the number of occupants and their locations within the monitored area.\cite{longoAccurateOccupancyEstimation2019, teissedre-2019}
```

File: report/src/presence_technologies/presence_intro.tex [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
+Several technologies can be utilized for detecting room occupancy, including infrared (IR) sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, ultrasonic sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, microwave sensors\cite{woodward-2021}, computer vision\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{OccupancySensorMotion}, Bluetooth and Wi-Fi\cite{teissedre-2019,longoAccurateOccupancyEstimation2019}, pressure sensors\cite{OccupancySensorMotion}, acoustic sensors\cite{OccupancySensorMotion}, and CO2 sensors\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{jinSensingProxyOccupancy2015}.\cite{faragherLocationFingerprintingBluetooth2015}\todo{Insert all the types you find and cite here. Remove TODO when this section is done.}
 Each technology has its strengths and weaknesses in terms of accuracy, cost, and performance.
 
+These technologies can be divided into various categories, but we will examine technologies in three main categories, \textbf{environmental}, \textbf{video-based}, and \textbf{radio-based}.
```
Comment (Pattrigue) [inline report/src/presence_technologies/presence_intro.tex:10]:
```suggestion
These technologies can be divided into various categories, but we will examine technologies in three main categories: \textbf{Environmental}, \textbf{video-based}, and \textbf{radio-based}.
```

File: report/src/bib/mybib.bib (hunk 1 above)
Comment (chhoumann) [reply report/src/bib/mybib.bib:417]: Good catch - it's just my Zotero filing system.

File: report/src/bib/mybib.bib (hunk 2 above)
Comment (chhoumann) [reply report/src/bib/mybib.bib]: Removed

File: report/src/presence_technologies/index.tex (hunk 3 above)
Comment (chhoumann) [reply report/src/presence_technologies/index.tex]: Yep good

**Review by Pattrigue — APPROVED** [review]

**Pull request merged**